edualgo/graph_algorithms/Dijkstra_algorithms.py

```diff
@@ -0,0 +1,112 @@
+from .__init__ import print_msg_box
+
+import sys
+
+
+def dijkstra_algorithm(graph, hint=False):
+
+    # Providing the graph
+    size = int(input())  # Size of the Vertices & Edges
+
+    vertices = []
+    for x in range(size):
+        vertices.append([int(y) for y in input().split()])
+
+    edges = []
+    for x in range(size):
+        edges.append([int(y) for y in input().split()])
+
+    # Find which vertex is to be visited next
+
+    global visited_and_distance
+
+    def to_be_visited(graph, hint=False):
```

Comment:
There is no need to assign functions inside functions, you can just take this function out of the `def dijkstra_algorithm(graph, hint=False):` and call it by passing the required parameters.

Remember ours is a learning library and we keep things clean and simple as much as possible.